src/main/groovy/au/org/ala/ws/security/authenticator/AlaOidcAuthenticator.groovy

```diff
@@ -150,20 +154,60 @@ class AlaOidcAuthenticator extends UserInfoOidcAuthenticator {
             } else {
                 cred.userProfile = generateAlaUserProfile(profile)
             }
+
+            if (accessTokenRoles) {
+                cred.userProfile.addRoles(accessTokenRoles)
+            }
```

Comment:
I think we need something similar to this to set up userprofile.permissons which are scopes.